src/local/butler/deploy.py

```diff
@@ -449,6 +451,27 @@ def _deploy_terraform(config_dir):
   common.execute(f'rm -rf {terraform_dir}/.terraform*')
 
 
+def _is_safe_deploy_day():
+  time_now_in_ny = now(pytz.timezone('America/New_York'))
+  day_now_in_ny = time_now_in_ny.weekday()
+  return day_now_in_ny not in {4, 5, 6}  # The days of the week are 0-indexed.
+
+
+def _enforce_safe_day_to_deploy():
```

Comment:
This should be called in execute(), right?

Reply:
Ha I knew I forgot something.